src/main/java/Event.java

```diff
@@ -0,0 +1,75 @@
+import java.time.LocalDateTime;
+
+/**
+ * Event class creates a type of Task called event which contains a date and description of the task
+ */
+public class Event extends Task {
+
+
+    private LocalDateTime date;
+
+    /**
+     * constructor to mkae Event
+     * @param description
+     * @param date
+     */
+    public Event(String description, LocalDateTime date) {
+        super(description);
+        this.date = date;
+    }
+    /**
```

Comment:
Maybe separate the method and javadoc? I know this is a check style issue, but I still feel that separate them makes more sense, so I suppress the warning. If I'm wrong, just correct me. Thanks.

Reply:
Separating is correct i think as logical units should be separated from one another